lms/static/js/learner_dashboard/models/course_enroll_model.js

```diff
@@ -0,0 +1,20 @@
+/**
+ *  Track optIn status and submit form
+ */
+;(function (define) {
+    'use strict';
+
+    define([
+            'backbone'
+        ],
+        function( Backbone) {
+            return Backbone.Model.extend({
+                defaults: {
+                    course_id: '',
+                    optIn: false,
+                },
+                urlRoot: '/api/commerce/v0/baskets/'
```

Comment:
It still strikes me as odd that we feel the need to hardcode this URL but not any others. You're now passing the URL through from the backed; maybe just use that?